README.md

```diff
@@ -51,7 +51,7 @@ The current documentation is fully automated at the moment (except writing the c
 4. cd oas-raml-converter
 5. npm install
 6. npm run build
-7. npm i -g swagger-to-slate
+7. npm i swagger-to-slate
```

Comment:
Are you sure swagger-to-slate is installed in `raml2markdown/node_modules/.bin/swagger-to-slate`? To me it looks like it goes to `raml2markdown/oas-raml-converter/node_modules/.bin/swagger-to-slate`.